File: src/main/java/net/matees/arcade/itemrush/ItemRushSettingsMenu.java
package net.matees.arcade.itemrush;

import me.kodysimpson.simpapi.menu.PlayerMenuUtility;
import net.matees.arcade.Minigame;
import net.matees.settings.SettingsMenu;

public class ItemRushSettingsMenu extends SettingsMenu {
    public ItemRushSettingsMenu(PlayerMenuUtility playerMenuUtility) {
        super(playerMenuUtility);
    }

    @Override
    public String getMenuName() {
        return "Item Rush Settings";
    }

    @Override
    public Minigame getMinigame() {
        return ItemRush.getInstance();
    }
}

File: src/main/java/net/matees/arcade/itemrush/ItemRushSettingsMenu.kt
package net.matees.arcade.itemrush

import me.kodysimpson.simpapi.menu.PlayerMenuUtility
import net.matees.arcade.Minigame
import net.matees.settings.AbstractSettingsMenu

class ItemRushSettingsMenu(playerMenuUtility: PlayerMenuUtility?) : AbstractSettingsMenu(playerMenuUtility) {
    override fun getMenuName(): String {
        return "Item Rush Settings"
    }

    override val minigame: Minigame
        get() = ItemRush.Companion.instance
}
